api/src/components/publicationVersion/service.ts

```diff
@@ -116,23 +132,45 @@ export const getById = (id: string) =>
         include: defaultPublicationVersionInclude
     });
 
-export const get = (publicationId: string, version: string | number) =>
+export const privateGetById = (id: string) =>
+    client.prisma.publicationVersion.findFirst({
+        where: {
+            id
+        },
+        include: privatePublicationVersionInclude
+    });
+
+const getVersionFilterFromStringOrNumber = (versionFilter: string | number) => ({
```

Comment:
Do we need to specify the type in the name? Just `getVersionFilter` should do really. Maybe also restrict it to just a string, you're doing impure functions in a way, the first condition is converting versionFilter to a number which is a hidden side effect (if I'm reading it right)

Reply:
Yep, don't see a reason why this shouldn't be simplified - now done.